hack/make-rules/test.sh

```diff
@@ -61,6 +61,7 @@ kube::test::find_dirs() {
           -o -path './third_party/*' \
           -o -path './staging/*' \
           -o -path './vendor/*' \
+          -o -path './test/e2e_node/services/*' \
```

Comment:
This is to omit that directory, you should add it to the next `find`.

Reply:
whoops, thanks!

Reply:
done